### ISSUE TITLE
Android arm64-v8a release build crashes when loading KTX files

### ISSUE THREAD

**Issue by atteneder** [atteneder/KtxUnity#11]
basis files work
development builds work

**Comment by atteneder** [comment]
Couldn't reproduce in 0.8.0

**Comment by ruben-ivre** [comment]
I know this issue is closed, but we are having the exact same issue on 1.0.0, arm64, KTX files only, release only, using your example project.

**Comment by atteneder** [comment]
Hi @ruben-ivre ,

sorry to hear that. I'll re-open the issue and investigate if you give me a bit more info:

- Which Unity version are you using?
- Is it reproducible 100% 
- Does it happen with a any KTX file or a particular one? If latter, can you provide it?
- Does it crash with single files or only in conjunction with loading a bunch of files?

thanks!

**Comment by ruben-ivre** [comment]
Thanks for the quick response. I've just used your example project and published it to android, no changes other than that. 

> * Which Unity version are you using?
2019.4.21
> * Is it reproducible 100%
Yes. Publish your example project to android, run it, choose "Format Test scene", pick any image and then any format in the right column, that should make it crash.
> * Does it happen with a any KTX file or a particular one? If latter, can you provide it?
I've used the ones included in your example project, and it crashed with all I tested with.
> * Does it crash with single files or only in conjunction with loading a bunch of files?
A single KTX file is enough to crash it.
